src/EditorFeatures/Core/Shared/Tagging/Utilities/TagSpanIntervalTree.cs

```diff
@@ -48,7 +54,7 @@ public bool HasSpanThatContains(SnapshotPoint point)
         var snapshot = point.Snapshot;
         Debug.Assert(snapshot.TextBuffer == _textBuffer);
 
-        return _tree.HasIntervalThatContains(point.Position, length: 0, new IntervalIntrospector(snapshot));
+        return _tree.HasIntervalThatContains(point.Position, length: 0, new IntervalIntrospector(snapshot, _spanTrackingMode));
```

Comment:
instead of the node needing to hold onto the tracing mode, we just pass it through with the introspector (a lightweight struct) when queryign the tree.